packages/cli/src/ceramic-daemon.ts

```diff
@@ -831,12 +834,12 @@ export class CeramicDaemon {
       return
     }
 
-    const withFieldIndices = req.query.withFieldIndices
-    if (withFieldIndices && withFieldIndices.toString().toLowerCase() == 'true') {
-      const indexedModels = await this.ceramic.admin.getIndexedModelsWithFieldIndices()
+    const withModelData = req.query.withModelData
+    if (withModelData && withModelData.toString().toLowerCase() == 'true') {
+      const indexedModels = await this.ceramic.admin.getIndexedModelData()
       res.json({
         models: indexedModels.map((idx) => idx.streamID.toString()),
```

Comment:
since the client has to explicitly request the new format with an argument, we know if we get a request with this argument then we must be speaking to a new client.  So then changing the output format should be safe.  Meaning I think we don't need to have both `models` and `modelData` and duplicate information in the response.  I think we can just change the structure of `models` to be an array of documents instead of streamids, and new clients will know to expect that.

EDIT: I guess there would be an issue if you had a new client talking to an old server that didn't know about the new argument.  At some point we should probably standardize on whether we expect clients or servers to be upgraded first and only support cross-version compatibility in a single upgrade direction to simplify the surface area of what we need to maintain backwards compatibility for.  CC @oed @bigs

I guess duplicating the information for now is fine then to avoid having to make that decision right now

Reply:
Given that we have a versioned endpoint, was planning on putting it under a v0. We can do another endpoint (/modelData?) and mark this one as deprecated, but that's a little weird with a versioned endpoint, rather than doing a /v1/models

Reply:
I think a `/modelData` endpoint would be fine and is probably simplest